config.json

```diff
@@ -1,1052 +1,1065 @@
 {
```

Comment:
Ran the configlet fmt as described in the documentation:
https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md#configuring-the-exercise
It seems the configlet wasn't run in quite a time. Or the formatter changed drastically... don't know right now.

Reply:
Hmm. The configlet does many changes to this file, and they seem out of the scope of this MR.

We should handle configlet changes in a new PR (it'll perfectly fine if you run it and create that PR).

So, I'd leave this file untouched, just modifications related to `nucleotide-count`. You can do those changes *compatible* with the new configlet. It'll be ok. But, I wouldn't change the rest of exercises configuration in the PR.

What do you think?

Reply:
Yeah, actually I did so in the JS track with a separate PR that got merged just recently. Bit this epiphany struck me too late for thes ecma track. ;-)
I'll splitt it.

Reply:
👍